src/credentials/wizards/templates.ts

```diff
@@ -11,30 +11,20 @@ import { getIdeProperties } from '../../shared/extensionUtilities'
 import { ProfileTemplateProvider } from './createProfile'
 import { createCommonButtons } from '../../shared/ui/buttons'
 import { credentialHelpUrl } from '../../shared/constants'
-
-// TODO: use this everywhere else
-export enum ProfileKey {
-    AccessKeyId = 'aws_access_key_id',
-    SecretKey = 'aws_secret_access_key',
-    Process = 'credential_process',
-}
+import { BaseCredentialsProfileData } from '../sharedCredentials'
+import { sharedCredentialProperties } from '../providers/sharedCredentialsProvider'
 
 function getTitle(profileName: string): string {
     return localize('AWS.title.createCredentialProfile', 'Creating new profile "{0}"', profileName)
 }
 
-interface StaticProfile {
```

Comment:
This is replaced by `BaseCredentialsProfileData` in this commit, we will be centralizing these types since they will have use outside of this file.